resources/headers

```diff
@@ -1113,156 +532,685 @@ x-myqee-system-time
+x-json
+x-litmus
+x-locking
+x-oc-mtime
+x-remote-addr
+x-request-signature
+x-ua-device
+x-update-range
+x-varnish
+x-wp-nonce
+auth
+brief
+chunk-size
+client
+download-attachment
+download-bz2
+download-e-headers-sent
+download-e-invalid-archive-type
+download-e-invalid-content-type
+download-e-invalid-file
+download-e-invalid-param
+download-e-invalid-request
+download-e-invalid-resource
+download-e-no-ext-mmagic
+download-e-no-ext-zlib
+download-inline
+download-tar
+download-tgz
+download-zip
+header-lf
+header-status-client-error
+header-status-informational
+header-status-redirect
+header-status-server-error
+header-status-successful
+https-from-lb
+meth-delete
+meth-head
+meth-post
+multipart-boundary
+originator
+php
+recipient
+request-error
+request-vars
+secretkey
+status-ok
+xauthorization
+x-codeception-codecoverage
+x-codeception-codecoverage-config
+x-codeception-codecoverage-debug
+x-codeception-codecoverage-suite
+x-csrf-token
+x-dokuwiki-do
+x-helpscout-signature
+x-nokia-bearer
+xonnection
+x-purpose
+xroxy-connection
+x-user
+bae-env-appid
+catalog-server
+cookie-path
+custom-header
+forwarded-for-ip
+meth-get
+meth-put
+opencart
+unless-modified-since
+www-address
+x-content-type
+x-hub-signature
+x-signature
+bae-env-addr-sql-ip
+bae-env-addr-sql-port
+cache-info
+client-error-cannot-access-local-file
+client-error-cannot-connect
+client-error-communication-failure
+client-error-invalid-parameters
+client-error-invalid-server-address
+client-error-no-error
+client-error-protocol-failure
+client-error-unspecified-error
+error-formatting-html
+lock-token
+onerror-continue
+onerror-die
+overwrite
+prefer
+shib-application-id
+x-fireloggerauth
+cookie-domain
+https
+meth-
+modauth
+port
+post
+read-state-begin
+read-state-body
+read-state-headers
+socket-connection-err
+str-match
+transport-err
+coming-from
+nl
+ua-pixels
+x-coming-from
+x-jphone-display
+x-up-devcap-screenpixels
+x-whatever
+appname
+proxy-port
+version
+x-forward-for
+proxy-user
+x-em-uid
+x-file-type
+bar
+proxy
+timeout
+referrer
+x-forwarded-ssl
+x-jphone-uid
+x-file-size
+accepted
+appcookie
+bad-gateway
+bae-env-addr-bcs
+conflict
+continue
+created
+expectation-failed
+failed-dependency
+gateway-time-out
+gone
+insufficient-storage
+internal-server-error
+length-required
+locked
+method-not-allowed
+moved-permanently
+moved-temporarily
+multiple-choices
+multi-status
+no-content
+non-authoritative
+not-acceptable
+not-extended
+not-implemented
+not-modified
+partial-content
+payment-required
+precondition-failed
+processing
+proxy-authentication-required
+range-not-satisfiable
+request-entity-too-large
+request-time-out
+request-uri-too-large
+reset-content
+see-other
+service-unavailable
+switching-protocols
+temporary-redirect
+unprocessable-entity
+unsupported-media-type
+upgrade-required
+use-proxy
+variant-also-varies
+version-not-supported
+x-operamini-phone
+bad-request
+forbidden
+unauthorized
+user-agent-via
+appversion
+not-found
+url-strip-
```

Comment:
we don't want any entries that might be regarded as malformed as they'll cause FPs